src/easyscience/fitting/available_minimizers.py

```diff
@@ -1,55 +1,65 @@
 import warnings
+from dataclasses import dataclass
 from enum import Enum
-from enum import auto
 
 import pkg_resources
 
-installed_packages = {pkg.key for pkg in pkg_resources.working_set}
+installed_packages = {pkg.key.lower() for pkg in pkg_resources.working_set}
```

Comment:
I don't think installed_packages is used anywhere

Reply:
Thanks, left over from old way of determining if a given module with minimizers is available